ext/opcache/jit/zend_jit.c

```diff
@@ -3294,12 +3320,6 @@ ZEND_EXT_API int zend_jit_op_array(zend_op_array *op_array, zend_script *script)
 		return zend_jit_setup_hot_trace_counters(op_array);
 	} else if (zend_jit_trigger == ZEND_JIT_ON_SCRIPT_LOAD) {
 		return zend_real_jit_func(op_array, script, NULL);
-	} else if (zend_jit_trigger == ZEND_JIT_ON_DOC_COMMENT) {
```

Comment:
Having a Jit attribute on the op_array should take precedence over all the other triggers. If you configure the ON_ATTRIBUTE trigger, then it **only** jits on attributes.